engine/common/net_encode.c

```diff
@@ -1289,8 +1311,16 @@ qboolean Delta_ReadField( sizebuf_t *msg, delta_t *pField, void *from, void *to,
 		if( bChanged )
 		{
 			iValue = MSG_ReadBitLong( msg, pField->bits, bSigned );
-			flValue = (int)iValue * ( 1.0f / pField->multiplier );
-			flValue = flValue * pField->post_multiplier;
+			if( bSigned )
```

Comment:
Can DT_FIELD even be unsigned?

Reply:
Yes it can
https://github.com/dreamstalker/rehlds/blob/aaffe43e3e7a99f267b29ed84d7405a989242052/rehlds/engine/delta.cpp#L968

Reply:
Yeah, indeed.